flow/utils/aimsun/load.py

```diff
@@ -132,7 +138,7 @@ def get_dict_from_objects(sections, nodes, turnings, cen_connections):
 model.load(template_path)
 
 # collect the simulation parameters
-params_file = 'flow/core/kernel/scenario/data.json'
+params_file = 'flow/core/kernel/network/data.json'
```

Comment:
In lines 177 and 186 we have similar bugs. Could you also fix them (replace 'scenario' with 'network')?

Reply:
In my local directory, those two files are still in `core/kernel/scenario`, not `network`. I don't think that they should be changed.

Reply:
yup, please do

Reply:
Done!

Reply:
great, thanks!